Xamarin.Forms.Core/IBorderElement.cs

```diff
@@ -1,11 +1,21 @@
-﻿namespace Xamarin.Forms
+﻿using System.ComponentModel;
+
+namespace Xamarin.Forms
 {
-	interface IBorderElement
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public interface IBorderElement
 	{
 		//note to implementor: implement this property publicly
 		Color BorderColor { get; }
+		int CornerRadius { get; }
+		Color BackgroundColor { get; }
+		double BorderWidth { get; }
 
 		//note to implementor: but implement the methods explicitly
 		void OnBorderColorPropertyChanged(Color oldValue, Color newValue);
+		bool IsSet(BindableProperty targetProperty);
```

Comment:
these explicit properties are used by BorderBackgroundManager and BorderDrawable
Not sure if there's a better way to surface these.  Maybe they should be moved to controller?

Reply:
let's discuss this one later today